src/features/wisdom/utils.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable tsdoc/syntax */
+import { Disposable, Event, EventEmitter } from "vscode";
```

Comment:
this file can be moved under `src/features/wisdom/utils/` folder and renamed to be more specifc eg. `promiseHandlers.ts`